lib/devise-otp.rb

```diff
@@ -82,8 +82,8 @@ module Devise
   #
   # email otp token if valid for
   #
-  mattr_accessor :otp_by_email_code_valid_for
-  @@otp_by_email_code_valid_for = 5.minutes
+  mattr_accessor :otp_by_email_code_timeout
+  @@otp_by_email_code_timeout = 5.minutes
```

Comment:
timeout is when you are not allowed to do something. I like the previous wording better.